apps/api/src/app/activity/dtos/workflow-runs-request.dto.ts

```diff
@@ -56,4 +58,8 @@ export class GetWorkflowRunsRequestDto {
   @IsOptional()
   @IsISO8601()
   createdLte?: string;
+
+  @IsOptional()
+  @IsEnumOrArray(SeverityLevelEnum)
+  severity?: SeverityLevelEnum[] | SeverityLevelEnum;
```

Comment:
allow filtering the activity feed by `severity`

Reply:
a thought, we could make it be only an array on the api layer, so the type is a bit simpler

Reply:
ahh you mean this? I can do that
```
@Transform(({ value }) => (Array.isArray(value) ? value : [value]))
```